<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompletionSettingsPage</class>
 <widget class="QWidget" name="CompletionSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Code Completion</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="caseSensitive">
        <property name="toolTip">
         <string>Do a case-sensitive match for completion items.</string>
        </property>
        <property name="text">
         <string>&amp;Case-sensitive completion</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="autoInsertBraces">
        <property name="toolTip">
         <string>Automatically insert (, ) and ; when appropriate.</string>
        </property>
        <property name="text">
         <string>&amp;Automatically insert braces</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="partiallyComplete">
        <property name="toolTip">
         <string>Insert the common prefix of available completion items.</string>
        </property>
        <property name="text">
         <string>Autocomplete common &amp;prefix</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
